app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -92,8 +92,7 @@ class SettingsFragment : PreferenceFragmentCompat(), SettingsView {
                 isValid = true
                 if (BiometricManager.from(requireActivity()).canAuthenticate() == BiometricManager.BIOMETRIC_SUCCESS) {
                     setLock = true
-                    authenticator.title = getString(R.string.biometric_set_title)
-                    authenticator.authenticate()
+                    authenticator.authenticate(requireContext().getString(R.string.biometric_set_title))
```

Comment:
authenticator.authenticate(getString(R.string.biometric_set_title)) as getString() is available at fragment

Reply:
ah good catch, force of habit here :)